internal/store/models/storedobject.go

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+// models describes the data types that will be used when storing export data.
+package models
+
+type storedObject struct {
+	// ID is the unique identifier for this record once stored.
+	ID string
+
+	// AppServiceKey identifies the app to which this data belongs.
+	AppServiceKey string
+
+	// Payload is the data to be exported
+	Payload []byte
+
+	// RetryCount is how many times this has tried to be exported
+	RetryCount int
+
+	// PipelinePosition is where to pickup in the pipeline
+	PipelinePosition int
+
+	// Version is a hash of the functions to know if the pipeline has changed.
+	Version string
+
+	// CorrelationID is an identifier provided by EdgeX to track this record as it moves
+	CorrelationID string
+
+	// EventID is a checksum used to identify an EdgeX event from the core services and mark it as pushed.
+	EventID string
```

Comment:
Add `EventChecksum` for when the input data from Core Data is CBOR. This is different than the EventID . Used for marked as pushed for CBOR data.